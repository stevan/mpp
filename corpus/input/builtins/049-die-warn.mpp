warn("Warning");
die("Fatal error");