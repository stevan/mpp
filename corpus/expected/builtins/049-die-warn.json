[
  {
    "type": "Warn",
    "message": {
      "type": "String",
      "value": "\"Warning\""
    }
  },
  {
    "type": "Die",
    "message": {
      "type": "String",
      "value": "\"Fatal error\""
    }
  }
]